kitipy/utils.py

```diff
@@ -139,7 +139,7 @@ def on_end():
 
 
 def append_cmd_flags(cmd: str, **kwargs) -> str:
-    """Build a command string by appending flags in \**kwargs.
+    """Build a command string by appending flags in **kwargs.
```

Comment:
Unfortunately this backslash is needed as this comment is interpreted as restructured text by sphinx and `**` is the marker for bold text :/

Reply:
Yep indeed :confused: Maybe a double backslash would do the job :thinking: